apps/checker/handlers/tcp.go

```diff
@@ -196,7 +207,7 @@ func (h Handler) TCPHandler(c *gin.Context) {
 		}
 	}
 
-	c.JSON(http.StatusOK, gin.H{"message": "ok"})
+	c.JSON(http.StatusOK, response)
```

Comment:
Same as above
Instead of dummy data return the actual result useful for the run endpoint